lib/oban/testing.ex

```diff
@@ -169,11 +169,11 @@ defmodule Oban.Testing do
     |> repo.one()
   end
 
-  defp available_jobs(repo) do
+  defp available_jobs(repo, opts) do
     base_query([])
-    |> select([:worker, :args])
+    |> select(^Keyword.keys(opts))
```

Comment:
Let's pull these out above the pipe chain and reference them in both places.

Reply:
sure :+1: